md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/AppDeploymentQueue.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.wipro.ats.bdre.md.beans.table;
+
+/**
+ * Created by SU324335 on 3/8/2016.
+ */
+public class AppDeploymentQueue {
+
+    private Long appDeploymentQueueId;
+    private Integer processId;
+    private String username;
+    private String appDomain;
+    private String appName;
+    private Short appDeploymentStatusId;
+    private Integer pageSize;
+    private Integer page;
+    private Integer counter;
+
+    public AppDeploymentQueue(){
+    }
+
+    public AppDeploymentQueue(Long appDeploymentQueueId, Integer processId, String username, String appDomain, String appName, Short appDeploymentQueueStatus, Integer pageSize, Integer page, Integer counter) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 9 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00107)